### ISSUE TITLE
Introspection fails, report id 1302: Unable to resolve scalar field

### ISSUE THREAD

**Issue by jgoux** [prisma/prisma2#1410]
Hello,

Here is my introspection error report : 

```
Introspection failed: Introspected schema can't be parsed.
Introspected Schema:

datasource db {
  provider = "postgresql"
  url      = "postgres://colisage:colisage@localhost:5432/colisage?schema=public"
}
model action {
  id    String @id
  label String @unique
}

model bag {
  barcode                  String                    @default("lpad((nextval(bag_barcode_seq::regclass)), 5, 0)") @unique
  barcode_comment          String?
  id                       String                    @id
  temperature_max          Float?
  temperature_min          Float?
  temperature_tag_id       String?                   @unique
  boxes                    box[]                     @relation(references: [bag_id])
  samples                  sample[]                  @relation(references: [bag_id])
  temperature_measurements temperature_measurement[] @relation(references: [bag_id])
}

model box {
  barcode         String   @default("lpad((nextval(box_barcode_seq::regclass)), 5, 0)") @unique
  barcode_comment String?
  capacity        Int
  id              String   @id
  bag_id          bag?
  destination_id  site     @relation("box_destination_idTosite")
  location_id     site?    @relation("box_location_idTosite")
  samples         sample[] @relation(references: [box_id])
}

model sample {
  barcode                         String                           @unique
  demand_executed_at              DateTime
  demand_number                   String
  id                              String                           @id
  nature                          String?
  patient                         String
  received_at                     DateTime?
  sampled_at                      DateTime?
  bag_id                          bag?
  box_id                          box?
  location_id                     site?                            @relation("sample_location_idTosite")
  site_id                         site?                            @relation("sample_site_idTosite")
  sample_events                   sample_event[]                   @relation(references: [sample_id])
  sample_temperature_measurements sample_temperature_measurement[] @relation(references: [sample_id])
}

model sample_event {
  created_at DateTime
  id         String   @id
  payload    String?
  type       String
  sample_id  sample
}

model sample_temperature_measurement {
  id                         String                  @id
  sample_id                  sample
  temperature_measurement_id temperature_measurement

  @@unique([sample_id, temperature_measurement_id], name: "sample_temperature_measuremen_sample_id_temperature_measure_key")
}

model site {
  hexalis_clabo                    String                    @unique
  id                               String                    @id
  label                            String                    @unique
  type                             String
  boxes_box_destination_idTosite   box[]                     @relation("box_destination_idTosite")
  boxes_box_location_idTosite      box[]                     @relation("box_location_idTosite")
  samples_sample_location_idTosite sample[]                  @relation("sample_location_idTosite")
  samples_sample_site_idTosite     sample[]                  @relation("sample_site_idTosite")
  temperature_measurements         temperature_measurement[] @relation(references: [site_id])
}

model temperature_measurement {
  alert                           Boolean
  alert_temperature_max           Float?
  alert_temperature_min           Float?
  alert_treated_at                DateTime?
  created_at                      DateTime
  id                              String                           @id
  measurements                    String
  alert_treated_by                user?                            @relation("temperature_measurement_alert_treated_byTouser")
  bag_id                          bag
  created_by                      user                             @relation("temperature_measurement_created_byTouser")
  site_id                         site
  sample_temperature_measurements sample_temperature_measurement[]
}

model user {
  email                                                                   String?
  id                                                                      String                    @id
  keycloak_id                                                             String                    @unique
  name                                                                    String?
  username                                                                String                    @unique
  temperature_measurements_temperature_measurement_alert_treated_byTouser temperature_measurement[] @relation("temperature_measurement_alert_treated_byTouser")
  temperature_measurements_temperature_measurement_created_byTouser       temperature_measurement[] @relation("temperature_measurement_created_byTouser")
}

Schema parsing
thread 'main' panicked at 'Unable to resolve scalar field 'sample_id'', src/libcore/o
ption.rs:1185:5
```

**Comment by janpio** [comment]
Possibly tracked in https://github.com/prisma/prisma2/issues/1377

**Comment by janpio** [comment]
This should be fixed with the current `alpha` @Simskii! (The fix will also be included in the upcoming preview release, `2.0.0-preview022`)
